app/views/councillor_contributions/_contribution_form_input.html.haml

```diff
@@ -7,3 +7,4 @@
       .suggested-councillor-input-group
         = suggested_councillor_field.label :email, class: "councillor-contribution-label"
         = suggested_councillor_field.text_field :email
+      = @councillor_contribution.suggested_councillors.last.errors.full_messages.to_sentence
```

Comment:
I added `flash[:error]` in the controller new action to inform something is not right, and used `Proc.new` to display different error messages for each case of blank email and invalid email format.